### DISCUSSION TITLE
Rotating the map with mouse & keyboard (web, desktop)

### DISCUSSION THREAD

**Discussion by TNorbury** [fleaflet/flutter_map#1511] [Q&A] [answered]
Is there a built in way to rotate the map with mouse and keyboard? For example on Google Earth I can rotate by holding down command/control and dragging around. Currently this doesn't appear to be supported.

If it's not currently supported, has anyone come up with a solution?

Package and flutter version:
flutter_map: `3.1.0`
Flutter: `3.7.12`

Thanks!

**Comment by JaffaKetchup** [comment, marked as answer]
UPDATE: This functionality was added by #1592, and is available since v6.

---

Hey @TNorbury,
This unfortunately isn't currently supported.

**Reply by JaffaKetchup** [reply]
Update: This functionality was added by #1592, available in v6.

**Reply by TNorbury** [reply]
Awesome, thanks for letting me know!
